javascript/builtins/intl/Intl.json

```
@@ -110,6 +110,61 @@
             }
           }
         },
+        "supportedValuesOf": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf",
+            "spec_url": "https://tc39.es/ecma402/#sec-intl.supportedvaluesof",
```

This is a little fussy, but… would you set this to the proposal URL after all?

The spec is in w3c/browser-specs now, so when dependabot picks up the release later today, I can merge it. Then I can rebase _this_ PR with the updated `package.json`, which will unbreak the tests and everything will be very tidy.

---

```suggestion
            "spec_url": "https://tc39.es/proposal-intl-enumeration/#sec-intl.supportedvaluesof",
```
@ddbeck Absolutely! Done - handed over to you now.